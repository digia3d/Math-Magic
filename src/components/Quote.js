import React from 'react';

import Header from './Header';

function Quote() {
  return (
    <>
      <Header />
      <div>
        <h3>Welcome to our page!</h3>
      </div>
      <section className="descr">
        <div>
          <p className="quote">
            Mathmatics is not about numbers,eauations, computations,
            or algorithms: it is about understanding.
            <span className="quspan">-William Paul Thurston</span>
          </p>
        </div>
      </section>
    </>
  );
}

export default Quote;
