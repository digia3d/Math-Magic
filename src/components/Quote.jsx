import React from 'react';

function Quote() {
  return (
    <>
      <div>
        <h3>Welcome to our page!</h3>
      </div>
      <section className="descr">
        <div>
          <p className="quote">
            Mathmatics is not about numbers,eauations, computations,
            or algorithms: it is about understanding.
            <span className="quspan">-William Paul Thurston</span>
          </p>
        </div>
      </section>
    </>
  );
}

export default Quote;
