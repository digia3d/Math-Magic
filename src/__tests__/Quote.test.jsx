import React from "react";
import renderer from 'react-test-renderer';
import Quote from "../components/Quote";
import '@testing-library/jest-dom';

describe('Testing Quote page', () => {
  it('test if Quote page is rendered', () => {
      const quoteTest = renderer
          .create(<Quote />)
          .toJSON();
      expect(quoteTest).toMatchSnapshot();
  });
});
