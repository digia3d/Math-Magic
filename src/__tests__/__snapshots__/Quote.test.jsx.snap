// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Quote page test if Quote page is rendered 1`] = `
Array [
  <div>
    <h3>
      Welcome to our page!
    </h3>
  </div>,
  <section
    className="descr"
  >
    <div>
      <p
        className="quote"
      >
        Mathmatics is not about numbers,eauations, computations, or algorithms: it is about understanding.
        <span
          className="quspan"
        >
          -William Paul Thurston
        </span>
      </p>
    </div>
  </section>,
]
`;
